pom.xml

```diff
@@ -8,7 +8,7 @@
         <relativePath />
     </parent>
     <artifactId>metadata-utils</artifactId>
-    <version>1.11-SNAPSHOT</version>
+    <version>1.12-SNAPSHOT</version>
```

Comment:
Why are you changing the snapshot version here?  This is done when we release.

Reply:
will nuke